project.clj

```diff
@@ -11,6 +11,7 @@
                                [:name "Elango Cheran"]
                                [:email "elango@google.com"]
                                [:timezone "-8"]]]
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [quil "2.2.5"]]
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [org.clojure/clojurescript "1.7.170"]
```

Comment:
I don't think lein-cljsbuild dep is required in the library. I currently require lein-cljsbuild only in my project (which has clojure-turtle as dependency), and the clojure-turtle dependency is build automatically.